pages/linux/xdg-mime.md

```diff
@@ -0,0 +1,16 @@
+# xdg-mime
+
+> Query and change desktop file type handling.
+> More information: <https://www.freedesktop.org/wiki/Software/xdg-utils/>.
+
+- View default application for opening filetype:
```

Comment:
I guess there should be a space between "file" and "type" ?

Reply:
The xdg manpage examples use `filetype` throughout which is why I kept to it.
```
...
xdg-mime query { filetype | default }
...
```

Reply:
I was also considering `mime/type` or `file/type` to show that 99% of the time its of that form?

Reply:
I think for the sake of new users, it might be helpful to slightly deviate from the man pages if it makes things slightly more clear.